### PR TITLE
adds train and test splitter with valid_pct

### PR DESCRIPTION
**Pull request by s-mostafa-a**

### CODE REVIEW

**Comment by jcreinhold** [comment]
Hi @s-mostafa-a. Thanks for the pull. I can't merge this PR as is. The first commit changes the setup.py incorrectly (for my repo). The second commit makes a significant number of cosmetic changes (I am guessing you ran a PEP-8 style checker on the file) which in general is fine, but they obscure the contribution.

Regarding the actual contribution, I think something like this would be ok to commit; however, the implementation could re-use code better. I changed the interface of NiftiDataset to accept a list of filenames, like your NiftiLightDataset (see 5b2d2a6adbed7da374b8c10ce4086cee6e44dfda). IMO, your NiftiTrainValidation class is better suited as a function (i.e., a function that returns a tuple of two NiftiDatasets where one corresponds to the training set and the other corresponds to the validation set).

If you update your contribution to reflect the above, I'll merge it.

**Comment by s-mostafa-a** [comment]
Ok Jacob Thanks, I close this pull request and I'll make another one with mentioned terms.
